packages/template-retail-react-app/app/routes.jsx

```diff
@@ -38,10 +38,10 @@ const Registration = loadable(() => import('./pages/registration'), {
 const ResetPassword = loadable(() => import('./pages/reset-password'), {fallback})
 const Account = loadable(() => import('./pages/account'), {fallback})
 const Cart = loadable(() => import('./pages/cart'), {fallback})
-const Checkout = loadable(() => import('./pages/checkout'), {
+const Checkout = loadable(() => import('./pages/checkout-container'), {
```

Comment:
I think I'd prefer the previous path. CheckoutContainer is not really a _page_ component.